pkg/cmd/cli/cmd/expose.go

```diff
@@ -13,15 +13,20 @@ import (
 )
 
 const (
-	exposeLong = `Take a replicated application and expose it as a route.
+	exposeLong = `Look up a service and expose it as a route.
```

Comment:
I would change this to "Expose containers internally as services or externally via routes"